cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -90,7 +90,7 @@ func (o *ReleaseOptions) Run() error {
 	if err != nil {
 		return err
 	}
-	if len(pkgConfigs.PkgMetadatas) != 1 || len(pkgConfigs.PkgMetadatas) != 1 {
+	if len(pkgConfigs.Pkgs) != 1 || len(pkgConfigs.PkgMetadatas) != 1 {
```

Comment:
Nice catch 🚀 
What do you think about making the error more descriptive, i.e telling exactly what is missing. So, if `len(pkgConfigs.Pkgs) != 1` then saying that "Reading package-resource.yml: expected exactly 1 Package, but got %v. (hint: delete the file and run `package init` again)"? (And same for PackageMetadata)

Reply:
That sounds like it could be work to update messaging for errors around the app. The error as is, does offer a suggestion on how it can be fixed. Perhaps an issue someone looking to start contributing can pick up as a good first issue?

As for the tests, I don't think I'll have time to get to them soon.

Reply:
Sure, no problem, we can make the change as part of a separate PR.